src/utils/rest-api.js

```diff
@@ -1169,17 +1169,20 @@ export function copyOrMoveModifications(
     studyUuid,
     targetNodeId,
     modificationToCutUuidList,
-    copyType
+    copyInfos
 ) {
-    console.info(copyType + ' modifications');
+    console.info(copyInfos.copyType + ' modifications');
     const cutAndPasteModificationUrl =
```

Comment:
rename `cutAndPasteModificationUrl` to `copyOrMoveModificationUrl`

Reply:
fixed